srcpkgs/meld/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'meld'
 pkgname=meld
-version=3.20.4
-revision=2
-build_style=python3-module
-configure_args="--disable-schemas-compile"
-hostmakedepends="pkg-config intltool python3-distro itstool
- gtk-update-icon-cache libxml2-python3"
-depends="python3-gobject gsettings-desktop-schemas gtksourceview
+version=3.22.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config itstool gettext appstream-glib glib-devel"
+makedepends="python3-devel gtk+3-devel gtksourceview4-devel python3-gobject-devel"
+depends="python3-gobject python3-cairo gsettings-desktop-schemas gtksourceview4
  desktop-file-utils hicolor-icon-theme"
+checkdepends="${depends} python3-pytest"
 short_desc="Visual diff and merge tool"
 maintainer="Helmut Pozimski <helmut@pozimski.eu>"
 license="GPL-2.0-or-later"
-homepage="http://meldmerge.org/"
+homepage="https://meld.app"
+changelog="https://gitlab.gnome.org/GNOME/meld/-/raw/main/NEWS"
 distfiles="${GNOME_SITE}/meld/${version%.*}/meld-${version}.tar.xz"
-checksum=f48e10eec606f687a87061e78668f6bb40e63e032175c4c7033636b65a157d13
+checksum=3fc107c98ef6e75358ffd2b0d14c85ddb48fe14a11e939a94322faaa8e90c40d
 
-post_install() {
-	rm $DESTDIR/usr/share/glib-2.0/schemas/gschemas.compiled \
-		$DESTDIR/usr/share/icons/hicolor/icon-theme.cache
+do_check() {
+	ninja -C build test
+	python3 -m pytest test
```

Comment:
```suggestion
post_check() {
	python3 -m pytest test
```
does this work? the default `do_check()` should work, probably

Reply:
yep, that works fine